### PR TITLE
Properly ack messages.

### PR DESCRIPTION
**Pull request by jmorton**
- Remove erroneous no_ack on a channel.
- Ensure channel only prefetches a single message so other workers have something to do.
- Fail fast... nack and die.

### CODE REVIEW

**Review by davidvhill — COMMENTED** [review]
a-ok

File: cw/__worker_main__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from cw import RABBIT_EXCHANGE
+from cw import RESULT_ROUTING_KEY
+import traceback
+import sys
+
+
+def main():
+    conn = None
+    try:
+        conn = open_connection(RABBIT_HOST, RABBIT_PORT)
```
Comment (cgaustin) [inline cw/__worker_main__.py:18]: we'll need to make the ssl option configurable here

File: cw/__worker_main__.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        listen(callback(conn, RABBIT_EXCHANGE, RESULT_ROUTING_KEY),
```
Comment (cgaustin) [inline cw/__worker_main__.py:19]: the callback func doesn't use the connection, did you have another idea for it there?

File: cw/worker.py [hunk 3]
```diff
@@ -186,7 +211,33 @@ def run(self, input_d):
...
+    out = dict()
+    for k, v in body.items():
+        out_k, out_v = k, v
+        if isinstance(k, bytes):
+            out_k = k.decode('utf-8')
+        if isinstance(v, bytes):
+            out_v = v.decode('utf-8')
+        out[out_k] = out_v
+    return out
+
```
Comment (cgaustin) [inline cw/worker.py:225]: mentioned above, connection not used in callback or handler

**Pull request merged**